tests/app/v2/notifications/test_post_notifications.py

```diff
@@ -243,65 +106,6 @@ def test_post_sms_notification_with_celery_persistence_returns_201_with_sms_send
     assert mocked.called
 
 
-def test_post_sms_notification_uses_sms_sender_id_reply_to(notify_api, client, sample_template_with_placeholders, mocker):
-    notify_api.config["FF_NOTIFICATION_CELERY_PERSISTENCE"] = False
-    sms_sender = create_service_sms_sender(service=sample_template_with_placeholders.service, sms_sender="6502532222")
-    mocked = mocker.patch("app.celery.provider_tasks.deliver_throttled_sms.apply_async")
-    data = {
-        "phone_number": "+16502532222",
-        "template_id": str(sample_template_with_placeholders.id),
-        "personalisation": {" Name": "Jo"},
-        "sms_sender_id": str(sms_sender.id),
-    }
-    auth_header = create_authorization_header(service_id=sample_template_with_placeholders.service_id)
-
-    response = client.post(
-        path="/v2/notifications/sms",
-        data=json.dumps(data),
-        headers=[("Content-Type", "application/json"), auth_header],
-    )
-    assert response.status_code == 201
-    resp_json = json.loads(response.get_data(as_text=True))
-    assert validate(resp_json, post_sms_response) == resp_json
-    assert resp_json["content"]["from_number"] == "+16502532222"
-    notifications = Notification.query.all()
-    assert len(notifications) == 1
-    assert notifications[0].reply_to_text == "+16502532222"
-    mocked.assert_called_once_with([resp_json["id"]], queue="send-throttled-sms-tasks")
-
-
-def test_notification_reply_to_text_is_original_value_if_sender_is_changed_after_post_notification(
```

Comment:
I wonder in which scenario was this test applying? We want to axe it because it will be too difficult to test with the additional celery work to save the notification?